bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BKException.java

```diff
@@ -142,6 +144,7 @@ public static BKException create(int code) {
         int UnauthorizedAccessException = -102;
         int UnclosedFragmentException = -103;
         int WriteOnReadOnlyBookieException = -104;
+        int LedgerIdOverflowException = -106;
```

Comment:
Part of our pushback. I wasn't sure how best to handle the discrepancy. Mixing them up now seems undesirable.